src/components/CardsContainer.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div id="cards-container">
+        <h1 class="title">{{ fullName() }} Tasks </h1>
+        <div id="main-cards-container">          
+            <div v-for="(category,index) in categories" :key="index" class="cards-section">
```

Comment:
pemotongan component masih kurang tepat, harusnya yang satu div lo v-for ini yang namanya CardContainer, dan CardContainer di looping disini, mungkin nama komponen yang sekarang gw komen ini MyKanbanBoard atau KanbanBoard